src/nlp/metric.py

```diff
@@ -228,18 +231,20 @@ def compute(self, predictions=None, references=None, timeout=120, **metrics_kwar
         predictions = self.data["predictions"]
         references = self.data["references"]
         with temp_seed(self.seed):
-            output = self._compute(predictions=predictions, references=references, **metrics_kwargs)
+            output = self._compute(predictions=predictions, references=references, **kwargs)
         return output
 
-    def add_batch(self, predictions=None, references=None, **kwargs):
-        """ Add a batch of predictions and references for the metric's stack.
+    def add_batch(self, *, predictions=None, references=None, **kwargs):
```

Comment:
Here also, maybe we should disallow `**kwargs` to avoid silent errors. It's a bit of a design decision because then it means we can't override `add_batch` in a derived class but I think we are probably fine with it.

Reply:
done